test/powershell/Modules/PowerShellGet/PowerShellGet.Tests.ps1

```diff
@@ -159,16 +156,17 @@ Describe "PowerShellGet - Module tests (Admin)" -Tags @('Feature', 'RequireAdmin
     }
 
     It "Should install a module correctly to the required location with AllUsers scope" {
-        Install-Module -Name $ContosoServer -Repository $RepositoryName -Scope AllUsers
+        Install-Module -Name $ContosoServer -Repository $RepositoryName -Scope AllUsers -AllowClobber
         $installedModuleInfo = Get-InstalledModule -Name $ContosoServer
 
         $installedModuleInfo | Should -Not -BeNullOrEmpty
         $installedModuleInfo.Name | Should -Be $ContosoServer
-        $installedModuleInfo.InstalledLocation.StartsWith($script:programFilesModulesPath, [System.StringComparison]::OrdinalIgnoreCase) | Should -BeTrue
+        # Not implemented yet
```

Comment:
Please add a link to an issue tracking this.

Reply:
https://github.com/PowerShell/PowerShellGet/issues/212